### PR TITLE
Add custom translation achievements

### PR DESCRIPTION
**Pull request by danielzhang130**
- Add "bilingual", "trilingual", "linguist" achievements
- Add CustomTranslationAchievementListener
- Update AchievementsCommand
- Using placeholder images
- Will fix users who already satisfy achievements in the next commit

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [ ] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [ ] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [x] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

### CODE REVIEW

File: src/Translation/CustomTranslationAchievementListener.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      }
+
+      $project = $this->program_manager->find($project_id);
+
+      if (null === $project) {
+        return;
+      }
+
+      try {
+        $this->achievement_manager->unlockAchievementCustomTranslation($project->getUser());
```
Comment (dmetzner) [inline src/Translation/CustomTranslationAchievementListener.php]: getUser could return null which then would result in a type error @runtime. Either directly catch the throwable or check first if the return value of getUser is null

File: src/Translation/CustomTranslationAchievementListener.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      } catch (Exception $e) {
+      }
```
Comment (dmetzner) [inline src/Translation/CustomTranslationAchievementListener.php:57]: I'd recommend at least to log a warning when catching all exceptions in such a generic way. Else, we might miss something that we shouldn't :)

**Comment by dmetzner** [comment]
Do you want me to merge & deploy your new achievements already with the placeholder images?

**Comment by danielzhang130** [comment]
> Do you want me to merge & deploy your new achievements already with the placeholder images?

As long as you think it's ok to use these images and update them in the future.

We think it could be something like chat bubbles
image

**Review by dmetzner — APPROVED** [review]

**Pull request merged**